src/render/mod.rs

```diff
@@ -654,13 +653,9 @@ impl Renderer {
         // prepare target and globals
         let (mx_inv_proj, mx_view, mx_vp) = {
             let p: [[f32; 4]; 4] = camera.matrix(self.aspect_ratio()).into();
-            let node = &hub.nodes[&camera.object.node];
-            let w = match node.scene_id {
-                Some(id) if Some(id) == scene_id => node.world_transform,
-                Some(_) => panic!("Camera does not belong to this scene"),
-                None => node.transform,
-            };
-            let mx_view = Matrix4::from(w.inverse_transform().unwrap());
+            let node = &hub[&camera];
+            let world_transform = node.transform; //TODO!!!
```

Comment:
What's this for?

Reply:
TODO: compute the world transform for the camera, which is expected here